src/functions/delimsizing.js

```diff
@@ -64,8 +64,8 @@ function checkDelimiter(
     } else {
         throw new ParseError(
             "Invalid delimiter: '" +
-            // $FlowFixMe, do not polyfill
-            (symDelim ? symDelim.text : JSON["stringify"](delim)) +
+            // eslint-disable-next-line transform-runtime-aliasing/no-transform-runtime-aliasing
+            (symDelim ? symDelim.text : JSON.stringify(delim)) +
```

Comment:
The reason why babel doesn't polypill this anymore is b/c we're using browserlist right?

Reply:
(The "polyfill" or "polyfilling" here refers to `core-js aliasing`)

@kevinbarabash Unfortunately, `@babel/plugin-transform-runtime` doesn't use Browserslist (yet), so it includes every polyfill even if all target environments support it or it isn't needed. So I think it's better to whitelist polyfills than blacklist them.

> After careful consideration:
> - Polyfills are generally expensive
> - ES modules of core-js@2 is not available, it needs to be wrapped in ES module to use in the ESM build (rollup)
> - We have removed codes that require polyfill in #1469, #1470, #1563, and #1591 
> - We added a warning (linter) against code that requires polyfill in #1597
> - Babel 7 separated out Babel's helpers from its polyfilling behavior in runtime, `transform-runtime` no longer polyfills by default

Reply:
Therefore, I've disabled automatic core-js aliasing, which is default in Babel 7, and changed to warn that it is not polyfilled and adding polyfill may be needed.